lib/results.c

```diff
@@ -93,6 +95,8 @@ const char *STDCALL snowflake_type_to_string(SF_DB_TYPE type) {
             return "TIME";
         case SF_DB_TYPE_BOOLEAN:
             return "BOOLEAN";
+        case SF_DB_TYPE_ANY:
+            return "ANY";
```

Comment:
I assume this will be set to the server and accepted?

Reply:
Yes this is sent to the server (and accepted) when the user chooses SF_C_TYPE_NULL as their input binding

Reply:
ok